Tests/test_file_avif.py

```diff
@@ -327,6 +327,7 @@ def test_exif_save(
             if orientation == 1:
                 assert "exif" not in reloaded.info
             else:
+                assert reloaded.getexif()[274] == orientation
```

Comment:
This assertion fails before this change for orientations 6 and 8.

Reply:
We can include this new assertion if you like, but actually, the next line
```python
assert reloaded.info["exif"] == exif_data
```
already fails with the new parameters. This is because the next line is checking the full EXIF data, not just the orientation.

Reply:
I realize they're redundant, but it was a bit obscure identifying the mismatch with the bytes, so I thought this more specific assertion might be more diagnostic of failures